src/NodeApi/Runtime/NodejsPlatform.cs

```diff
@@ -25,25 +25,39 @@ public sealed class NodejsPlatform : IDisposable
     /// <summary>
     /// Initializes the Node.js platform.
     /// </summary>
-    /// <param name="libnodePath">Path to the `libnode` shared library, including extension.</param>
+    /// <param name="libnode">
+    /// Name of the `libnode` shared library.
+    /// Has to be a full file path when using .NET Framework.
+    /// </param>
     /// <param name="args">Optional application arguments.</param>
     /// <param name="execArgs">Optional platform options.</param>
     /// <exception cref="InvalidOperationException">A Node.js platform instance has already been
     /// loaded in the current process.</exception>
     public NodejsPlatform(
-        string libnodePath,
+        string libnode,
         string[]? args = null,
         string[]? execArgs = null)
     {
-        if (string.IsNullOrEmpty(libnodePath)) throw new ArgumentNullException(nameof(libnodePath));
-
         if (Current != null)
         {
             throw new InvalidOperationException(
                 "Only one Node.js platform instance per process is allowed.");
         }
 
-        nint libnodeHandle = NativeLibrary.Load(libnodePath);
+#if NET7_0_OR_GREATER
+        var entryAssembly = System.Reflection.Assembly.GetEntryAssembly();
```

Comment:
Even using AssemblyDirectory, you still need to pass an assembly. I'd rather use the system default but I can change the assembly to get the current assembly instead of the entry.